### PR TITLE
8322278: Generational ZGC: Adjust the comment of ZHeuristics::use_per_cpu_shared_small_pages

### PR DESCRIPTION
**Pull request by lgxbslgx**
Hi all,

This trivial patch fixes the method `ZHeuristics::use_per_cpu_shared_small_pages` to meet its comment.

Thanks for the review.

Best Regards,
-- Guoxiong

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8322278](https://bugs.openjdk.org/browse/JDK-8322278): Generational ZGC: Adjust the comment of ZHeuristics::use_per_cpu_shared_small_pages (**Enhancement** - P5)

### Reviewers
 * [Erik Österlund](https://openjdk.org/census#eosterlund) (@fisk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/17138/head:pull/17138` \
`$ git checkout pull/17138`

Update a local copy of the PR: \
`$ git checkout pull/17138` \
`$ git pull https://git.openjdk.org/jdk.git pull/17138/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17138`

View PR using the GUI difftool: \
`$ git pr show -t 17138`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/17138.diff">https://git.openjdk.org/jdk/pull/17138.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/17138#issuecomment-1860204632)

### CODE REVIEW

File: src/hotspot/share/gc/z/zHeuristics.cpp
Comment (kimbarrett) [inline src/hotspot/share/gc/z/zHeuristics.cpp:66]: The proposed change just reverts what appears to have been an intentional code
change in JDK-8307058. That change should have updated the comment.  So I
don't think this change should be made.  The comment should be fixed instead.

**Review by kimbarrett — CHANGES_REQUESTED** [review]

File: src/hotspot/share/gc/z/zHeuristics.cpp
Comment (fisk) [reply src/hotspot/share/gc/z/zHeuristics.cpp:66]: Agreed.

Comment (lgxbslgx) [reply src/hotspot/share/gc/z/zHeuristics.cpp:66]: Should I change the title of this issue and fix the comment in this patch? Or open another issue and PR to fix it?

Comment (lgxbslgx) [reply src/hotspot/share/gc/z/zHeuristics.cpp]: I update the code which only changes the comment, and revise the title.

**Review by kimbarrett — COMMENTED** [review]
This looks okay to me, but I don't really know this part of ZGC.  So don't count me
as a reviewer.

**Review by fisk — APPROVED** [review]
Looks good.

**Comment by lgxbslgx** [comment]
@fisk @kimbarrett  Thanks for your reviews. Do you think this patch is trivial? If not, I will wait for another review.

**Comment by fisk** [comment]
It looks trivial; ship it!

**Pull request closed without merging**
